packages/core/src/app/customer/PasswordField.tsx

```diff
@@ -35,23 +29,7 @@ const PasswordField: FunctionComponent<PasswordFieldProps> = ({
         <TranslatedString id={ 'customer.password_label' } />
     ), []);
 
-    const footer = useMemo(() => {
-        if (!forgotPasswordUrl) {
-            return null;
-        }
-
-        return <a
-            data-test="forgot-password-link"
-            href={ forgotPasswordUrl }
-            rel="noopener noreferrer"
-            target="_blank"
-        >
-            <TranslatedString id="customer.forgot_password_action" />
-        </a>;
```

Comment:
"Forgot password" is only used in the login form. Therefore, it would be better to place it there inside the same `<p>`, which also contains "send login link", "create an account".